core/livepeernode.go

```diff
@@ -78,14 +79,16 @@ type LivepeerNode struct {
 //NewLivepeerNode creates a new Livepeer Node. Eth can be nil.
 func NewLivepeerNode(e eth.LivepeerEthClient, wd string, dbh *common.DB) (*LivepeerNode, error) {
 	rand.Seed(time.Now().UnixNano())
-	return &LivepeerNode{
+	lpNode := &LivepeerNode{
 		Eth:          e,
 		WorkDir:      wd,
 		Database:     dbh,
 		EthServices:  make(map[string]eth.EventService),
 		SegmentChans: make(map[ManifestID]SegmentChan),
 		segmentMutex: &sync.RWMutex{},
-	}, nil
+	}
+	lpNode.Accounts.Balances = make(map[ManifestID]balance)
```

Comment:
Can we create a constructor for `Accounts` that handles initialization of any internal fields instead of directly initializing the fields of `Accounts` in this manner?